# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/gfx-backend-metal/shaders/macros.h
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/gfx-backend-metal/shaders/macros.h b/third_party/rust/gfx-backend-metal/shaders/macros.h
deleted file mode 100644
--- a/third_party/rust/gfx-backend-metal/shaders/macros.h
+++ /dev/null
@@ -1,5 +0,0 @@
-#ifdef __METAL_MACOS__
-#   define GFX_RENDER_TARGET_ARRAY_INDEX [[render_target_array_index]]
-#else
-#   define GFX_RENDER_TARGET_ARRAY_INDEX
-#endif